File: cron.php
<?php

include_once("init.php");

// Select all entries for today's date
$sql = "SELECT * FROM StudyConcepts WHERE date = DATE(NOW())";
$result = $conn->query($sql);

if ($result->num_rows > 0) {
    $ch = curl_init();
    while ($row = $result->fetch_assoc()) {
        // Concat all our data together
        $data = "name=Study For " . $row['milestone'] . "&desc=Concepts: " . $row['concept'] . "&due=null&idList=$TRELLO_LIST&token=$TRELLO_TOKEN&key=$TRELLO_KEY";

        // Config curl
        $headers = array();
        $headers[] = "Content-Type: application/x-www-form-urlencoded";
        curl_setopt($ch, CURLOPT_URL, "https://api.trello.com/1/cards");
        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
        curl_setopt($ch, CURLOPT_POSTFIELDS, "$data");
        curl_setopt($ch, CURLOPT_POST, 1);
        curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);

        // Execute curl
        curl_exec($ch);

        // Handle errors
        if (curl_errno($ch)) {
            echo 'Error:' . curl_error($ch);
        }
    }
}

File: cron.py
import mysql.connector
import requests
import configparser

config = configparser.ConfigParser()
config.read('config.ini')

mydb = mysql.connector.connect(
  host=config['ss']['DB_IP'].strip('\"'),
  user=config['ss']['DB_USER'].strip('\"'),
  passwd=config['ss']['DB_PASS'].strip('\"'),
  database=config['ss']['DB_DB'].strip('\"'),
  charset="ascii",
  auth_plugin='mysql_native_password'
)

mycursor = mydb.cursor()

mycursor.execute('SELECT StudyTests.testName, concept FROM StudyConcepts, StudyTests WHERE StudyConcepts.date = DATE(NOW()) AND testID = StudyTests.id')

myresult = mycursor.fetchall()

key=config['ss']['TRELLO_KEY'].strip('\"')
token=config['ss']['TRELLO_TOKEN'].strip('\"')

for x in myresult:
    url = "https://api.trello.com/1/cards"
    data = {
        "name": "Study for " + x[0],
        "desc": x[1],
        "idList":config['ss']['TRELLO_LIST'].strip('\"'),
        "idLabels":"5b65f7740e84690feeab06f8",
        "key" : key,
        "token" : token
    }
    requests.post(url, data = data)
